### PR TITLE
[no-ticket] bump workbenchGoogle2 for new google-cloud-firestore version

### PR DESCRIPTION
**Pull request by ehigham**
Ticket: <Link to Jira ticket>
<Put notes here to help reviewer understand this PR>

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've filled out the [Security Risk Assessment](https://broadinstitute.github.io/dsp-appsec-security-risk-assessment/) and attached the result to the JIRA ticket

### CODE REVIEW

**Review by andy7i — APPROVED** [review]
LGTM, assuming tests pass and we use the super official libs from https://github.com/broadinstitute/workbench-libs/pull/963

**Comment by ehigham** [comment]
jenkins retest

**Review by gpolumbo-broad — APPROVED** [review]

File: automation/project/Dependencies.scala [hunk 1]
```diff
@@ -8,7 +8,7 @@ object Dependencies {
   val akkaHttpV = "10.2.2"
 
   val workbenchGoogleV = "0.21-ae11b9f"
-  val workbenchGoogle2V = "0.23-7ddf186"
+  val workbenchGoogle2V = "0.24-541cef41-SNAP"
```
Comment (ehigham) [inline automation/project/Dependencies.scala]:
```suggestion
  val workbenchGoogle2V = "0.24-447afa5"
```

File: project/Dependencies.scala [hunk 2]
```diff
@@ -12,10 +12,10 @@ object Dependencies {
   val http4sVersion = "0.21.13"
 
   val workbenchUtilV   = "0.6-74c9fc2"
-  val workbenchUtil2V   = "0.1-74c9fc2"
+  val workbenchUtil2V   = "0.2-541cef41-SNAP"
```
Comment (ehigham) [inline project/Dependencies.scala]:
```suggestion
  val workbenchUtil2V   = "0.2-447afa5"
```

File: project/Dependencies.scala [hunk 3]
```diff
@@ -12,10 +12,10 @@ object Dependencies {
...
   val workbenchModelV  = "0.15-f9f0d4c"
   val workbenchGoogleV = "0.21-8ce5b9b"
-  val workbenchGoogle2V = "0.23-11a45ad"
+  val workbenchGoogle2V = "0.24-541cef41-SNAP"
```
Comment (ehigham) [inline project/Dependencies.scala]:
```suggestion
  val workbenchGoogle2V = "0.24-447afa5"
```

**Comment by ehigham** [comment]
jenkins retest

**Pull request merged**